.circleci/config.yml

```diff
@@ -182,6 +186,24 @@ jobs:
                 name: coveralls
                 command: npm run coveralls
 
+  check-coding-style:
+    docker:
+      - image: cimg/node:current
+    steps:
+      - show-npm-version
+      - checkout
+      - shellcheck/install
+      - install-dependencies:
+          cache-id: solc-js
+      - run:
+          name: Check for javascript/typescript coding style
+          command: npm run lint
+      - shellcheck/check:
+          ignore-dirs: |
+            .git/
+            node_modules/
+            dist/
```

Comment:
Maybe shellcheck does need the `./` bit after all? I see in the CI output that it did not skip files inside node_modules:

```
./node_modules/tape/example/static/build.sh
./node_modules/foreground-child/changelog.sh
No ShellCheck Errors Found
```
Unless it just prints them without checking?

Reply:
By the way, the orb is nice. I see that it for example automatically adds `shellcheck.log` as an artifact (if non-empty). That's handy.

Reply:
oh no! Indeed it is not skipping those directories. There is an example of the config in the official repo: https://github.com/CircleCI-Public/shellcheck-orb/blob/73e2427de6b0a5d0c27527b608ee62880e3dbfb0/.circleci/config.yml#L22 and the `./` is used. I will put it back.

Reply:
Hum...still it shows. So yeah, it seems that it parses anyway but probably don't check as you mentioned.

Reply:
Can you try it out locally on a file where you intentionally make a mistake to make sure?

Reply:
Yes. It seems that it indeed considers the "ignored" folders when they terminate with `/` (e.g. `node_modules/`). If I remove the slash in the end, it works.

With `node_modules/` or `./node_modules/`:
```
./scripts/wrong.sh
./node_modules/foreground-child/changelog.sh
./node_modules/tape/example/static/build.sh
ShellCheck Errors Found

In ./scripts/wrong.sh line 5:
if [ $1 -eq "1"]; then
^-- SC1009: The mentioned syntax error was in this if expression.
   ^-- SC1073: Couldn't parse this test expression. Fix to allow more checks.
                ^-- SC1020: You need a space before the ].
                ^-- SC1072: Missing space before ]. Fix any mentioned problems and try again.
```

Without the slash in the end `./node_modules`:
```
./scripts/wrong.sh
ShellCheck Errors Found

In ./scripts/wrong.sh line 5:
if [ $1 -eq "1"]; then
^-- SC1009: The mentioned syntax error was in this if expression.
   ^-- SC1073: Couldn't parse this test expression. Fix to allow more checks.
                ^-- SC1020: You need a space before the ].
                ^-- SC1072: Missing space before ]. Fix any mentioned problems and try again
```

In conclusion, we need the current directory prefix, and we should not add the end slash.

Reply:
ok. Let's revert to the way it was originally.

Too bad that it's that picky. In the shell these would be perfectly fine.

Reply:
Yes. I reverted it already. I opened an issue in the orb repo anyway, asking about this behavior.